lib/bootstrapper.js

```diff
@@ -0,0 +1,210 @@
+// Copyright IBM Corp. 2014,2016. All Rights Reserved.
+// Node module: loopback-boot
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+var assert = require('assert');
+var async = require('async');
+var utils = require('./utils');
+var path = require('path');
+var pluginLoader = require('./plugin-loader');
+var debug = require('debug')('loopback:boot:bootstrapper');
+var Promise = global.Promise || require('bluebird');
```

Comment:
In LoopBack 3.x, we are always using `bluebird` as our promise implementation. I am proposing to do the same in loopback-boot for consistency.

```js
var Promise = require('bluebird');
```